testing/iprange/APKBUILD

```diff
@@ -0,0 +1,56 @@
+# Contributor: Jonny Tyers <jtyers@gmail.com>
+# Maintainer: Jonny Tyers <jtyers@gmail.com>
+pkgname=iprange
+pkgver=1.0.3
+pkgrel=0
+pkgdesc="manages IP ranges (part of FireHOL)"
+url="https://github.com/firehol/iprange"
+arch="all"
+license="GPL2"
+depends=""
+depends_dev=""
+makedepends="$depends_dev"
```

Comment:
There’s no -dev subpackage, so variable `depends_dev` should not be declared and used. abuild even warns about this…